torch/csrc/utils/python_arg_parser.h

```diff
@@ -785,4 +796,19 @@ auto handle_torch_function_getter(THPVariable* self, const std::string& property
 // Used for setters of Tensor properties.
 auto handle_torch_function_setter(THPVariable* self, const std::string& property_name, PyObject* value) -> int;
 
+/*
+ * TODO: we could fuse this function with the 'append_overloaded_arg' in
+ * python_arg_parser.cpp in python args cases. Currently this function is
+ * needed by supporting __torch_function__ override in C++ custom ops.
+ *
+ * Check if the input arg supports __torch_function__, if so, append it to
+ * overloaded_args. If arg is a list or tuple, further check whether each
+ * element in list/tuple supports override.
+ *
+ * 'arg': the input argument to be checked
+ * 'overloaded_args': the vector to append the overloaded args.
+ *
+ */
+void extract_torch_function_overloaded_arg(PyObject* arg, std::vector<py::handle>& overloaded_args);
```

Comment:
Instead of taking the vector in by mutable reference, it will be clearer to take it by pointer, so you get the visual clue at the call site that this vector is going to get mutated.

Reply:
Sure, i just kept it consistent with append_overloaded_arg defined in python_arg_parser.cpp.